js/dataSets.js

```diff
@@ -180,3 +187,90 @@ function getMagPhyxColor(event) {
             return 'black';
     }
 }
+
+function parseKeystrokeData(eventData) {
+    let simulations = [];
+    let eventTypes = ['+input', '+delete', 'RUN', 'paste', 'cut', 'setValue', 'TASK', 'SUBMIT', 'undo', 'drag'];
+
+    let onlyFile = eventData[0];
```

Comment:
My laptop has a hard time running a dataset this large. 
I've been using let onlyFile = eventData[0].slice(x, x+10000) where x is some arbitrary index.

Reply:
Yeah mine has been struggling with the dataset as well. I might have a way to whittle it down but I'm not sure.